logstash-core-event-java/src/main/java/com/logstash/KeyNode.java

```diff
@@ -39,4 +39,20 @@ public String evaluate(Event event) throws IOException {
             return "%{" + this.key + "}";
         }
     }
+
+    // TODO: (colin) this should be moved somewhere else to make it reusable
+    //   this is a quick fix to compile on JDK7 a not use String.join that is
+    //   only available in JDK8
+    public static String join(List<?> list, String delim) {
```

Comment:
yeah, comment says that. will you force me to do it now? ;) it's just that I think we should probably do a namespace cleanup at some point and I didn't want to commit to a Utils class (which I really dislike in general because they often end-up as a bag of everything...)